crates/typst/src/layout/align.rs

```diff
@@ -412,20 +424,29 @@ cast! {
     VAlignment,
     self => Alignment::V(self).into_value(),
     align: Alignment => match align {
-        Alignment::V(v) => v,
+        Alignment::V(y) => y,
         v => bail!("expected `top`, `horizon`, or `bottom`, found {}", v.repr()),
     }
 }
 
-/// A vertical alignment which only allows `top` and `bottom`, thus excluding
-/// `horizon`.
+/// A vertical alignment which only allows either ends, thus excluding `horizon`.
```

Comment:
I personally think this change was unnecessary (and made it less clear). The type name already suggests we only allow outer values; the doc comment lists them for further clarity.

Reply:
I feel itchy just because whenever I hover above a reference of this type name I can already see its variants are Top and Bottom, so repeating the variant names again in the doc feels unnecessary.

But no strong opinion here.